src/sigmatcher/cli.py

```diff
@@ -140,16 +141,27 @@ def _read_definitions(signatures: list[Path]) -> tuple[ClassDefinition, ...]:
     return merge_definitions_groups(definition_groups)
 
 
+def _get_apktool_version(apktool: str) -> str:
+    proc = subprocess.run([apktool, "--version"], check=True, capture_output=True)
+    if proc.stderr:
+        # apktool 2.12.0 specifically has no --version flag.
+        # apktool>=2.12.1 and apktool<2.12.0 don't have this problem. See: https://github.com/iBotPeaches/Apktool/issues/3909
+        # When called with --version, it exits with code 0, and outputs "Unrecognized command: --version" to stderr.
+        proc = subprocess.run([apktool, "version"], check=True, capture_output=True)
```

Comment:
I think it's fine to ignore this special case and only support `--version`. i.e. only <2.12.0,>=2.12.1.